app/src/main/res/layout/activity_monitoring_group_page.xml

```diff
@@ -34,30 +34,31 @@
             <LinearLayout
                 android:layout_width="match_parent"
                 android:layout_height="wrap_content"
+                android:layout_marginTop="10dp"
                 android:orientation="horizontal">
 
                 <Button
```

Comment:
android lint is complaining: ButtonStyle: Buttons in button bars should be borderless; use `style="?android:attr/buttonBarButtonStyle"` (and `?android:attr/buttonBarStyle` on the parent)